compliance/azure/instances_without_fnm_agent/azure_instances_not_running_flexnet_inventory_agent.pt

```diff
@@ -218,14 +221,35 @@ end
 policy "policy_long_running_instances" do
   validate $ds_formatted_instances do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{len data }} is missing in FlexNet Manager."
-    detail_template <<-EOS
-| Subscription Name | VM Name | Resource Name | Location | VM Size | OS Type | Tags |
-| ----------------- | ------- |-------------- | -------- | ------- | ------- | ---- |
-{{ range data -}}
-| {{.subscriptionName}} | {{.name}} | {{.resourcename}} | {{.location}} | {{.vmSize }} | {{.osType}} | {{.tags}} |
-{{ end -}}
-EOS
     escalate $report_instances
     check eq(size(data),0)
+    export do
+      resource_level true
+      field "subscriptionName" do
+        label "Subscription Name"
+      end
+      field "id" do
+        label "VM Name"
+        path "name"
+      end
+      field "resourcename" do
+        label "Resource Name"
+      end
+      field "location" do
+        label "Location"
+      end
+      field "vmSize" do
+        label "VM Size"
+      end
+      field "osType" do
+        label "OS Type"
+      end
+      field "tags" do
+        label "Tags"
+      end
+      field "id" do
+        label "ID"
```

Comment:
This can be removed, id is already used above.

Reply:
made the change